### PR TITLE
feat(movies): configure create endpoint

### PR DESCRIPTION
**Pull request by jekatlob**
Why: We need a way to add movies to the database

What: creates an endpoint at `/movies` that accepts POST requests for record creation

Details: doesn't do much just yet.

### CODE REVIEW

**Comment by javillanueva1** [comment]
Please add a description like one found in the code review guidelines https://www.notion.so/lob/Code-Review-Guidelines-aad42d64cb724f3cadd1ab233bfc93df

File: lib/server.js [hunk 1]
```diff
@@ -12,4 +12,14 @@ const server = new Hapi.Server({
 
 server.connection({ port: Config.PORT });
 
+server.register([
+  require('hapi-bookshelf-serializer'),
```
Comment (sjl2) [inline lib/server.js]: nit: since this isn't used yet, no real need to add it yet.

Comment (jekatlob) [reply lib/server.js]: I'm just doing what the tutorial says!

**Comment by jekatlob** [comment]
Thanks for the example on the PR subject... I'm still trying to fully comprehend how they get used.

**Pull request merged**